docs/changelog/changelog.rst

```diff
@@ -40,6 +43,11 @@ Changes
   :class:`~holodeck.weather.WeatherController`
   (`#196 <https://github.com/BYU-PCCL/holodeck/issues/196>`_,
   `#263 <https://github.com/BYU-PCCL/holodeck/issues/263>`_)
+- Calling :meth:`~holodeck.environments.HolodeckEnvironment.send_world_command`
```

Comment:
Do we need to include this? AFAIK we don't encourage users to use this method, its kind of internal

Reply:
And this would be a bug fix IMO

Reply:
[it doesn't seem like we do](https://www.google.com/search?client=ubuntu&hs=wP4&channel=fs&ei=T-pXXpGmFqyV0PEPrvGX6Ao&q=send_world_command+site%3Aholodeck.readthedocs.io&oq=send_world_command+site%3Aholodeck.readthedocs.io&gs_l=psy-ab.3...2274.2522..4360...1.0..0.103.290.2j1......0....2j1..gws-wiz.k2KXSE6Azqk&ved=0ahUKEwjRiabYj_LnAhWsCjQIHa74Ba0Q4dUDCAo&uact=5)

Reply:
I included it mostly because it affects the weather commands. Calling them in certain worlds will now cause a crash, so I think its useful to document.